### ISSUE TITLE
how to test pre-trained model?

### ISSUE THREAD

**Issue by gnujimmik** [tensorflow/models#458]
## Please let us know which model this issue is about (specify the top-level directory)

I was wondering if I could feed an image not the chunk of images and get the accuracy from a pre-trained model. In the eval_image_classifier.py, I need to feed a directory of image data and got the only one result of accuracy. Anybody knows how to deal with this with each image, please?

**Comment by tatatodd** [comment]
@gnujimmik You can use the code in eval_image_classifier.py (https://github.com/tensorflow/models/blob/master/slim/eval_image_classifier.py) as a starting-point, and modify it to accomplish your goal.

Also note that we primarily use github issues to track bugs and installation problems. This is a question better suited for StackOverflow. Please ask it there and tag it with the `tensorflow` tag. Thanks!
